test/test-pages/tmz-1/expected.html

```diff
@@ -1,6 +1,7 @@
 <div id="readability-page-1" class="page">
     <div class="post single-post" id="post-2015_02_26_lupita-nyongo-pearl-dress-stolen-oscars">
         <p class="headline">
+            <h2 class="hf1">Lupita Nyong'o</h2>
```

Comment:
Maybe file a followup to remove things with class/id containing `headline`?

In fact, this `h2` is clearly a substring of the title in the metadata, so why isn't it being removed?